controllers/shooter.py

```diff
@@ -81,12 +97,13 @@ def searching(self) -> None:
             self.next_state("tracking")
         else:
             # TODO test this
-            # pose: wpilib.geometry.Pose2d = self.chassis.get_pose()
-            # rel: wpilib.geometry.Pose2d = self.TARGET_POSITION.relativeTo(pose)
-            # rel_heading = rel.rotation().radians()
-            # self.turret.scan(rel_heading)
-
-            self.turret.scan(0)  # TODO remove this
+            pose: wpilib.geometry.Pose2d = self.chassis.get_pose()
+            rel: wpilib.geometry.Pose2d = pose - self.TARGET_POSITION
```

Comment:
I think this type hint is incorrect - the difference between two Pose2ds should be a Transform2d (since you would add a Pose2d and Transform2d to get a new Pose2d).